psqlextra/backend/schema.py

```diff
@@ -75,6 +75,7 @@ class PostgresSchemaEditor(SchemaEditor):
     sql_add_range_partition = (
         "CREATE TABLE %s PARTITION OF %s FOR VALUES FROM (%s) TO (%s)"
     )
+    sql_detach_partition = "ALTER TABLE %s DETACH PARTITION %s"
```

Comment:
they don't add semicolon at the end of statements?

Reply:
no need. i did not put any to remain consistent with other statements in the file